app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
Yeah, what I meant that the prefix on the name is an non issue.

I still can't agree with the "image" suffix. Seems like the Style guide doesn't either:

"All other conventions still hold using a suffix of the type" and I can't find a mention of other suffixes.
- https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#feature-file-names

Angular way seems to be kebab case for file names. So that would make it `background.directive.js` or `background-image.directive.js`, but I don't think that's so important right now. Even the "ancestor" of this directive, the HTML attribute background applies an image as the background, if such is given to it.